doc/self-host.md

```diff
@@ -0,0 +1,31 @@
+# Hosting a Gateway
+
+## build
+```shell
+npm run build
+```
+
+## start
+The server needs at least the following environment variables:
+
+| variable                  | example                                   | comments            |
+|---------------------------|-------------------------------------------|---------------------|
+| M_LD_GATEWAY_AUTH__KEY    | `rootacc.keyid:0gcsqgsib3dqebaquaa4gna`   | Root API access key |
+| M_LD_GATEWAY_KEY__PUBLIC  | `MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKB...` | Root public key     |
+| M_LD_GATEWAY_KEY__PRIVATE | `MIIC3TBXBgkqhkiG9w0BBQ0wSjApBgkqhkiG...`  | Root private key    |
```

Comment:
**question:** What are these two used for?

Reply:
Signing JWTs, and Gateway operations if signatures are enabled. I made a call to have them be mandatory, whether or not you're using these features, to simplify the code. BTW, there's a utility for generating all this fluff, which I'll document in the next PR.